UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml

```diff
@@ -21,8 +21,7 @@
 			<skia:SKSwapChainPanel
 				x:Name="swapChain"
 				PaintSurface="OnPaintSwapChain"
-				PointerMoved="{x:Bind OnPointerMoved}"
-				Background="Transparent"
```

Comment:
You must keep the transparent background, otherwise on some platforms the pointer movements will not work.

Reply:
I was the sampe reason, the stable version wasn't accepting this property. The preview does. Now it's restored.